ergebnismeldung/src/main/java/com/fb/dlqpattern/ergebnismeldung/service/SendInfoService.java

```diff
@@ -18,7 +18,13 @@ public SendInfoService(SendInfoMapper sendInfoMapper) {
         this.sendInfoMapper = sendInfoMapper;
         log.debug("Nachricht erhalten");
     }
-    public SendInfoDTO getInfoDTOFromInfo(SendInfo sendInfo) {
-        return sendInfoMapper.infoToInfoDTO(sendInfo);
+
+    public void  getInfoDTOFromInfo(SendInfoDTO sendInfoDTO) {
```

Comment:
Der Methodenname lautet verkehrt herum angesichts dessen was er macht. Er nimmt aus InfoDTO Daten und bildet damit ein Info-Objekt. Somt soll die Methode heißen getInfoFromInfoDTO.
Außerdem, der Präfix "get" suggeriert, dass die Methode etwas zurückgibt, aber sie ist void. Ich würde sie "sendNachricht" oder ähnlich so lieber benennen. Wenn "EAI" fertig ist, sollte aus dieser Methode auch an diese geschickt werden.

Reply:
renamed to SendInfoFromDTO.